src/libumf.def

```diff
@@ -119,6 +119,9 @@ EXPORTS
     umfScalablePoolParamsSetKeepAllMemory
 ; Added in UMF_0.11
     umfCUDAMemoryProviderParamsSetAllocFlags
+    umfCtlExec
```

Comment:
please move CTL functions to 0.12 namespace

Reply:
also, bump OPS structure version to 0.12

Reply:
I've updated structures for pool and provider, and also def+map files.